.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
suggestion: use 3.7 ... 3.10 (3.6 is EOL)

Reply:
Done!